ui/src/views/Clients/ViewClients/index.jsx

```diff
@@ -119,15 +119,19 @@ export default class ViewClients extends PureComponent {
     const {
       classes,
       description,
+      location,
       data: { loading, error, clients },
     } = this.props;
+    const searchQuery = qs.parse(location.search.slice(1));
```

Comment:
`parse` is already defined on line 4. We could just use it directly and remove  `qs` from both here and line 4.